plonetheme/onegovbear/browser/z3cjbot/Products.TinyMCE.skins.tinymce.plugins.plonebrowser.plonebrowser.htm.pt

```diff
@@ -0,0 +1,309 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" i18n:domain="tinymce"
+    tal:define="portal_url context/portal_url; portal_tinymce context/portal_tinymce;">
+<head>
+  <title i18n:domain="plone.tinymce" i18n:translate="">Insert/Edit</title>
+  <script type="text/javascript" src="../../tiny_mce_popup.js"></script>
+  <script type="text/javascript" src="../../utils/mctabs.js"></script>
+  <script type="text/javascript" src="../../utils/form_utils.js"></script>
+  <script type="text/javascript" src="js/plonebrowser.js"></script>
+  <link rel="stylesheet" type="text/css" href="css/plonebrowser.css" />
+  <link rel="stylesheet" type="text/css" tal:attributes="href string:${portal_url}/portal_css/columns.css" />
+  <link rel="stylesheet" type="text/css" href="css/plonebrowser.css" />
+  <!-- PATCH START -->
+  <link rel="stylesheet" type="text/css" tal:attributes="href string:${portal_url}/theming.css" />
```

Comment:
This kind breaks caching. We probably should generate the URL with the helper function (in `ftw.theming.browser.theming_css`), that means we must make it traversable.. :confused: